### ISSUE TITLE
Provided TLS key file is not found upon restart

### ISSUE THREAD

**Issue by bsinou** [pydio/cells#391]
We have been reported that launching Cells with Docker, using environment variables to provide custom TLS cert and key is broken.
It works OK during the installation step but exposes a self-signed certificate rather than the provided one after restart.

You can observe this behavior with such a config: 
```
...
cells:
    image: pydio/cells:latest
    restart: unless-stopped
    ports: ["443:443"]
    environment:
      - CELLS_BIND=:443
      - CELLS_EXTERNAL=https://files.example.com
      - CELLS_TLS_CERT_FILE=/var/certs/files.example.com.crt
      - CELLS_TLS_KEY_FILE=/var/certs/files.example.com.key
    volumes: 
      - /var/certs/provided:/var/certs
...
```

After investigation, it appears that we have a typo in the code. It has been fixed in the coming v4 branch but the fix is not yet reported in a released hotfix for the v3 branch.  
Until it is, there is a _not-very-elegant-yet-working_ workaround. You have to define a second variable that points toward the **same** key file:

 ```
 ...
      - CELLS_TLS_CERT_FILE=/var/certs/files.example.com.crt
      - CELLS_TLS_KEY_FILE=/var/certs/files.example.com.key
      # Add this
      - CELLS_TLS_CERT_KEY=/var/certs/files.example.com.key
 ...
 ```
 and you are good to go.

**Comment by vmatekole** [comment]
@bsinou this is not related to your issue but since you are making a fresh install(3.05) like me, wondering if you have ran into similar issue #392  when using encryption at rest?

**Comment by cdujeu** [comment]
Solved in 3.0.7
